synthtool/py_templating_instructions/README.md

```diff
@@ -0,0 +1,32 @@
+# Using Synthtool for Python Samples
+
+Synthtool supports template files using [Jinja](http://jinja.pocoo.org/).
+
+Templates used for the Python samples are found in subdirectories of [`synthtool/gcp/templates/python_samples`](gcp/templates/python_samples)
+
+To use Synthtool for Python samples you will need to create or update two files in the root directory of the product folder to which the samples belong.
```

Comment:
Maybe more step-by-step:

Add `synth.py` and `.repo-metadata.json` to your product or sample directory if not already present.

```
cp synth-template.py PRODUCT_DIR/synth.py
cp .repo-metadata-template.json PRODUCT_DIR/.repo-metadata.json
```

Verify `synth.py` contains call to `gcp.CommonTemplates.py_samples()`

Update `.repo-metadata.json` `"samples"` array with information about the samples in the directory.

New READMEs will be generated every 24 hours